dijet/tasks/base.py

```diff
@@ -41,6 +41,9 @@ class HistogramsBaseTask(
     # Add nested sibling directories to output path
     output_collection_cls = law.NestedSiblingFileCollection
 
+    # Category ID for methods
+    category_id = {"sm": 1, "fe": 2}
```

Comment:
Might be good to use a different convention for class-level variables that are not parameters. Maybe use all-caps?
```suggestion
    LOOKUP_CATEGORY_ID = {"sm": 1, "fe": 2}
```